### ISSUE TITLE
consider running wasm binaries through wabt/wasm-strip

### ISSUE THREAD

**Issue by bketelsen** [deislabs/krustlet#178]
in build process to reduce size dramatically

**Comment by VishnuJin** [comment]
hello @bacongobbler 
is this enhancement about building and using wasm-strip on [demos](https://github.com/deislabs/krustlet/tree/master/demos) apps and updating demos documents ?

I would like to work on this, some suggestions or pointers from you on this topic would be valuable for me.
Thanks

**Comment by VishnuJin** [comment]
Can I work on this enhancement please ? 
I'm new to rust and open source.
I would really appreciate some more detailed pointers like what is expected from this enhancement.

**Comment by kurious-diru** [comment]
Hey, @thomastaylor312 @bacongobbler 

is this issue up for grabs? I am pretty new to this repo and would like it to give it a try, looks like @VishnuJin already tried an approach here, thanks!

**Comment by thomastaylor312** [comment]
Yeah! I don't think @VishnuJin is actively working on this. So feel free to take it @kurious-diru!

**Comment by VishnuJin** [comment]
@kurious-diru please do
